cloud-aws-native/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/resource/network/AwsStackUtil.java

```diff
@@ -0,0 +1,43 @@
+package com.sequenceiq.cloudbreak.cloud.aws.resource.network;
+
+import org.springframework.stereotype.Component;
+
+import com.amazonaws.services.ec2.model.DescribeVpcsRequest;
+import com.amazonaws.services.ec2.model.DescribeVpcsResult;
+import com.sequenceiq.cloudbreak.cloud.aws.common.client.AmazonEc2Client;
+import com.sequenceiq.cloudbreak.cloud.model.Network;
+
+@Component
+public class AwsStackUtil {
+
+    public static final String VPC_ID = "vpcId";
+
+    public static final String SUBNET_ID = "subnetId";
+
+    public static final String NO_PUBLIC_IP = "noPublicIp";
+
+    public boolean isExistingNetwork(AmazonEc2Client amazonEc2Client, Network network) {
+        DescribeVpcsResult result = amazonEc2Client.describeVpcs(new DescribeVpcsRequest().withVpcIds(network.getStringParameter(VPC_ID)));
```

Comment:
I think exception handling would be nice here as AWS throw an exception with some kind of `NotFound` error code when a resource does not exist in most of the cases.

Reply:
done